File: JobSearchingWebApp/Endpoints/Kompanija/GetById/KompanijaGetByIdResponse.cs
namespace JobSearchingWebApp.Endpoints.Kompanija.GetById
{
    public class KompanijaGetByIdResponse
    {
        public string Id { get; set; }
        public string Naziv { get; set; }
        public int GodinaOsnivanja { get; set; }
        public string Lokacija { get; set; }
        public string? Logo { get; set; }
        public string BrojZaposlenih { get; set; }
        public string KratkiOpis { get; set; }
        public string Opis { get; set; }
        public string? Website { get; set; }
        public string? LinkedIn { get; set; }
        public string? Twitter { get; set; }
        public int BrojOtvorenihPozicija { get; set; }
    }
}

File: front-end/src/app/endpoints/kompanija-endpoint/get-by-id/kompanija-get-by-id-response.ts
export interface KompanijaGetByIdResponse {
  id: string
  naziv: string
  godinaOsnivanja: number
  lokacija: string
  logo: string | null
  brojZaposlenih: string
  kratkiOpis: string
  opis: string
  website: string | null
  linkedIn: string | null
  twitter: string | null
  brojOtvorenihPozicija: number
  userName: string
  email: string
}
